src/test/java/LoginTests.java

```diff
@@ -13,7 +13,7 @@ public static void LoginEmptyEmailPasswordTest () {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://testpro.io/";
+        String url = "https://bbb.testpro.io/";
         //KBS edit
         driver.get(url);
         Assert.assertEquals(driver.getCurrentUrl(), url);
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void loginEmptyEmailPasswordTest ()`

![image](https://user-images.githubusercontent.com/104761482/209428163-6311d74d-be0e-481c-8572-f70201317e43.png)